content/download/debian.adoc

```diff
@@ -6,63 +6,194 @@ weight = 10
 :icons: fonts
 :iconsdir: /img/icons/
 
-== Debian Stable (Stretch)
+== Basic Information
 
-{{< repology debian_stable_backports >}}
+The information further down about the available KiCad packages might not
+always be fully up2date and correct due ongoing development. For getting an
```

Comment:
Could we use "updated" or "up to date" instead of "up2date" here?

Reply:
Agreed. Changed within next force-push.